### DISCUSSION TITLE
how to set value for the new added row using changeCellValue method

### DISCUSSION THREAD

**Discussion by allenlim-0627** [bosskmk/pluto_grid#181] [Q&A]
Hi, 

I tried to set a value for one of the cells that is a double type value within a newly added row that was created by using appendNewRows method. When a selection type value within the same new added row was selected, for example, "Fruit Apple" was selected, I want to populate the price value in the cell named "Price" with the method changeCellValue(key, value). I keep hitting the error saying that "NoSuchMethodError: Class 'double' has no instance getter 'key'.". 

The changeCellValue method is working fine in my other dart class, but the class is already populated plutoRows that I need for the plutoGrid.

Could you advise me on how to set the value on the appendNewRows? Thank you so much for the effort on the plutogrid and it's great tool to populate data. :)

**Comment by bosskmk** [comment]
I want to understand your question and try to reproduce the problem, but it's difficult.

Could you please provide me with minimal code to reproduce the problem?

Or can you provide me with steps to reproduce the problem so I can try it?

**Reply by allenlim-0627** [reply]
Hi bosskmk, I just look into my code one more time, it's solely my mistake of taking value incorrectly, but once again thank you for the response :)
